.github/workflows/model_tools.yml

```diff
@@ -0,0 +1,28 @@
+name: Model tools
+
+on:
+  pull_request:
+    branches: '**'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.8"
+      - name: Pip install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install openvino openvino-dev[caffe,mxnet,onnx,tensorflow2] -r tools/model_tools/requirements.in -r tools/model_tools/requirements-pytorch.in -r tools/model_tools/requirements-tensorflow.in
+      - name: Test representative-models.lst
+        run: |
+          python tools/model_tools/downloader.py --list tools/model_tools/tests/representative-models.lst -o . -j 4
```

Comment:
do we will use this job for preparing model cache or it will be separated build?
What about enabling sanity check that model not only can be downloaded and converted, but also infered? (we used benchmark app before for that if I right remember)

Reply:
We won't prepare model cache for now
perf_validation.py was disabled on old precommits as well with the message "Added fixes for 2022.1 package." :)

I'm not going to re-enable this now